python/docs/source/index.rst

```diff
@@ -17,58 +17,148 @@
 
 .. PySpark documentation master file
 
-=====================
-PySpark Documentation
-=====================
+=================
+PySpark Overview
+=================
 
-|binder|_ | `GitHub <https://github.com/apache/spark>`_ | `Issues <https://issues.apache.org/jira/projects/SPARK/issues>`_ | |examples|_ | `Community <https://spark.apache.org/community.html>`_
-
-PySpark is an interface for Apache Spark in Python. It not only allows you to write
-Spark applications using Python APIs, but also provides the PySpark shell for
-interactively analyzing your data in a distributed environment. PySpark supports most
-of Spark's features such as Spark SQL, DataFrame, Streaming, MLlib
-(Machine Learning) and Spark Core.
+**Date**: |today| **Version**: |release|
 
-.. image:: ../../../docs/img/pyspark-components.png
```

Comment:
I think we can remove `pyspark-components.png` in this repo.

Reply:
I'll remove it